src/main/java/io/swagger/swaggerhub/plugin/SwaggerHubUpload.java

```diff
@@ -66,6 +66,8 @@
     private String definitionFileNameRegex;
     @Parameter(property = "upload.uploadType", required = true)
     private String uploadType;
+    @Parameter(property = "upload.basepath", required = true)
+    private String basepath;
```

Comment:
I don't think this should be required. They should only add this if required. SaaS users would need to add this and presumably leave it empty. I'm surprised this didn't break tests as test resources don't have this property included.

Reply:
Good catch. Thats copy and paste for you. Will look into why this did not break tests though, it should of. If not will try to add a test that verify this.

Reply:
There's no try. Only do. 😉